test/Dialect/Calyx/emit.mlir

```diff
@@ -63,6 +63,8 @@ calyx.program {
       }
       // CHECK:   c0.go = c1.out;
       calyx.assign %c0.go = %c1.out : i1
+      // CHECK:   %done = 1'd1;
+      calyx.assign  %done = %c1 : i1
```

Comment:
Nit: The Calyx native well-formedness check will complain that this component uses both a `done` condition and a control program.

If possible, test this behavior in the components above.

Reply:
Can you open an issue about this? Seems like something we should verify.